image/tarexport/load.go

```diff
@@ -161,6 +151,59 @@ func (l *tarexporter) Load(inTar io.ReadCloser, outStream io.Writer, quiet bool)
 	return nil
 }
 
+func (l *tarexporter) Loader(tmpDir string, outStream io.Writer, progressOutput progress.Output) (loader, error) {
+	// check and try to load an OCI image layout
+	ociLayoutPath, err := safePath(tmpDir, "oci-layout")
+	if err != nil {
+		return nil, err
+	}
+	ociLayoutFile, err := os.Open(ociLayoutPath)
+	if err == nil {
```

Comment:
I feel like doing an `os.Stat` here would be a bit nicer -- so something like:

```
ociLayoutFi, err := os.Stat(ociLayoutPath)
if err == nil { ... }
```

Reply:
Yeah, I can change to that

Reply:
fixed